### DISCUSSION TITLE
Ден ось таку штуку нещодавно запостив FYI

### DISCUSSION THREAD

**Discussion by webknjaz** [ALERTua/air_raid_threat_reporter#15] [General]
https://news.ycombinator.com/item?id=40479729

**Comment by ALERTua** [comment]
ага, дякую, бачив. але там string parsing, це розраховано на дуже чітке формулювання повідомлень від каналів. мені це здається не дуже надійним :) особливо такі як Ніколаєвський Ваньок, які можуть написати: "ВСЕ ГОРИТЬ ВСЕ ЛЕТИТЬ ХАВАЙСЯ", і нічо не запарсиш з такого меседжа гггг
я ж намагався створити Modelfile з інструкціями моделі, щоб вона сама це фільтрувала.
але воно потребує ще годин і годин відпрацьовування інструкцій. але для цього потрібно, щоб сервер з відяхою був постійно увімкнений, а це не дуже реалістично поки що, витрачати ~140Wh при відключеннях на постійний ран LLM. так що поки на паузі.

**Reply by ALERTua** [reply]
але я канєша буду онукам розповідати, шо сам Вебкнязь коментив мої галімі поробки на гітхабі)) моя повага, Святе, мені дуже шкода було, що Варгеймінг не віднісся до тебе з повагою, на яку ти заслуговуєш. для мене ти — еталон. 🙏

**Reply by webknjaz** [reply]
Дякую за теплі слова :) Останнім часом дивуюсь, в скількох контекстах мене люди пам'ятають..

Мені тоді пропонували щось інше, але не моє трохи було. Сам я до того часу уже не дуже по іграх був. Мене тільки пайтоном затягнули були до вас, а потім воно закінчилось і і пішов шукати пригод інде.. Втім, було весело і спогади чудові!

**Reply by webknjaz** [reply]
До речі, ось тут Ден колись зробив чатик домашніх автоматизаторів, якщо ти іще не там: https://t.me/selfhostedua
